internal/alb/lb/loadbalancer.go

```diff
@@ -178,13 +179,17 @@ func NewCurrentLoadBalancer(o *NewCurrentLoadBalancerOptions) (newLoadBalancer *
 	}
 
 	// Check WAF
-	webACLResult, err := albwafregional.WAFRegionalsvc.GetWebACLSummary(o.LoadBalancer.LoadBalancerArn)
-	if err != nil {
-		return newLoadBalancer, fmt.Errorf("Failed to get associated Web ACL. Error: %s", err.Error())
-	}
 	var webACLId *string
-	if webACLResult != nil {
-		webACLId = webACLResult.WebACLId
+
+	if albendpoints.IsWAFRegionalAvailable(*aws.Config{}.Region) {
```

Comment:
I'm not really convinced about this way of getting current region.
If someone can give me advice here, it will be really appreciate.

Reply:
was speaking with @M00nF1sh about this kind of thing for the VPC id, i'm thinking the region would be a similar global config variable that we would use